qiskit/test/base.py

```diff
@@ -48,7 +41,9 @@
 # If testtools is installed use that as a (mostly) drop in replacement for
 # unittest's TestCase. This will enable the fixtures used for capturing stdout
 # stderr, and pylogging to attach the output to stestr's result stream.
-if HAS_FIXTURES:
+if _optionals.HAS_FIXTURES:
+    _optionals.HAS_TESTTOOLS.require_now("build test suite with fixtures")
```

Comment:
Does this error if you have fixtures installed but not testtools? I don't think this will ever come up because testtools is a requirement of fixtures: https://github.com/testing-cabal/fixtures/blob/master/requirements.txt (it's actually a bidirectional dependency much to the annoyance of linux package maintainers everywhere: https://github.com/testing-cabal/testtools/blob/master/requirements.txt#L5 ). But I'm not sure it would be correct even if it did come up because if we're missing fixtures or testtools we shouldn't error we just shouldn't be using the class with fixtures.